WordPress/Classes/ViewRelated/Reader/ReaderHelpers.swift

```diff
@@ -276,11 +292,31 @@ extension ReaderHelpers {
             }
             return true
         }
+
+        guard !mutableItems.isEmpty else {
```

Comment:
Should this guard just move up to the top? Sorting an empty list won't do anything anyway.